fe/app/components/Quote/QuoteElement.tsx

```diff
@@ -0,0 +1,34 @@
+import { QuoteElementProps } from "./Quote.models";
+
+const QuoteElement = ({
+  container,
+  text = {
+    content: <></>,
```

Comment:
Remove the initial state, the content prop is required

Reply:
you're right! thanks